packages/algob/src/internal/cli/cli.ts

```diff
@@ -82,6 +84,11 @@ async function main (): Promise<void> {
     const ctx = BuilderContext.createBuilderContext();
     const config = loadConfigAndTasks(runtimeArgs);
 
+    let taskName = parsedTaskName ?? TASK_HELP;
+    if (runtimeArgs.help && taskName !== TASK_HELP) {
+      taskName = TASK_HELP;
```

Comment:
This can break how help task is used for help of a specific command:
`algob help init` -> should print help about `init` task.
And this should work too:
`algob -h init` -> should print help about `init` task.
And this (but it's covered by 2nd case):
`algob init -h` -> should print help about `init` task.

Reply:
you are right. will fix it.